source/slang/slang-parameter-binding.cpp

```diff
@@ -1190,20 +1190,31 @@ static void addExplicitParameterBindings_GLSL(
     // If inference is not enabled for this kind, we can issue a warning
     if (!hlslToVulkanLayoutOptions->canInfer(vulkanKind, hlslInfo.space))
```

Comment:
Due to your change at line 1154, `hlslToVulkanLayoutOptions` can be `nullptr` here.